### DISCUSSION TITLE
PBPK Modeling of Buprenorphine in Adults, Children and Preterm Neonates

### DISCUSSION THREAD

**Discussion by LkKov** [Open-Systems-Pharmacology/Forum#734] [Announcement]
Dear OSP-community,

We have recently been working on PBPK modeling of buprenorphine in pediatrics. First, an adult PBPK model has been thoroughly developed, which is able to predict buprenorphine and norbuprenorphine venous and arterial blood plasma concentration-time profiles after intravenous administration. The model has been used for the assessment of buprenorphine drug-drug interactions with clarithromycin, itraconazole as well as rifampicin.
Subsequently, the adult PBPK model has been scaled to both a child and a preterm neonate population for predictions of individual plasma concentration-time profiles.

The corresponding manuscript, including comprehensive documentation, has been published online in Pharmaceutics. The PK-Sim model files presenting the PBPK models including simulations of clinical studies can be found [here](https://github.com/Open-Systems-Pharmacology/Buprenorphine-Model/).

Related publication:
Kovar L, Schräpel C, Selzer D, Kohl Y, Bals R, Schwab M, Lehr T. Physiologically-Based Pharmacokinetic (PBPK) Modeling of Buprenorphine in Adults, Children and Preterm Neonates. Pharmaceutics. 2020; 12(6):578. https://www.mdpi.com/1999-4923/12/6/578

Best regards,
Lukas

**Comment by StephanSchaller** [comment]
Congratulations, Lukas! Well done!

Please also post here: https://www.linkedin.com/groups/13527823/

AND here: https://www.linkedin.com/groups/2593589/

;-)
